tox.ini

```diff
@@ -11,10 +11,13 @@ deps =
     pytest-custom_exit_code
     prettytable
     black
+    pika
```

Comment:
nit:
maybe at some point we'll have to build a `requirements.txt`, and or a proper series of requirements for setuptools, including different optional modules.

In this case, pika/requests might probably go into the trigger-tests environment, but I think it can be done later.

Reply:
I'm a bit torn on this one: on one hand it would be nice only include pika & requests where required. On the other hand, it means that if they are missing from the default venv, then any tooling launched via `tox -e vevn -- $cmd` is not aware of them and these modules are not available for unit testing. I found the later to outweigh the former and thus opted for it.

Reply:
It was a nit, so it's no big deal.

I think it would be better if I show you what I mean with a follow-up PR, and we discuss it there.

Reply:
sure, sounds good to me.